fixtures/Program.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using Microsoft.CodeAnalysis;
```

Comment:
Is this used?

Reply:
Yes, some interface types are from this package.